Dockerfile

```
@@ -3,6 +3,9 @@ ARG NGINX_NAME="nginx-${NGINX_VERSION}"
 
 FROM debian AS build
 
+HEALTHCHECK --interval=30s --timeout=3s --start-period=60s \
+  CMD (curl -f http://localhost/) || exit 1
```

HTTP GET / responds with a 302 Moved Temporarily to Location: https://strn.network - which responds with 403. I think this is... "suboptimal", for several reasons:

A. Healthcheck on a URL which redirects, and externally at that not just internally, isn't clearly defined. E.g. on K8s using `GET /` as a [Liveness, Readiness and/or Startup Probe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/) leads to [a confusing weird `ProbeWarning`](https://github.com/kubernetes/website/issues/30849) - I just ran into it, with this container... 🥲  (I can't remember whether a container's `HEALTHCHECK` automagically become a probe on K8s, but even if it doesn't, another check would be be better, IMHO.)

B. It's not clear to me yet if just Nginx responding on / really means the container is actually fully ready.

@DiegoRBaquero in #124 mentions _"Could be used to test the test CID locally"_ and I just dug a bit more into what he may have meant by that... how about we instead propose using this:

```suggestion
  CMD (curl -f http://localhost/ipfs/QmXjYBY478Cno4jzdCcPy4NcJYFrwHZ51xaCP8vUwN9MGm/) || exit 1
```